### PR TITLE
docs: update docs to include plugin repositories

### LINKED ISSUE #1536: [Docs] mention other plugin indices on plugin-page
https://plugins.drone.io/

https://drone-plugin-index.geekdocs.de/plugins/drone-matrix/

### PR DESCRIPTION
**Pull request by hay-kot**
Closes #1536 

Not sure if you were looking for more here since the issue is pretty lean. Happy to add more or rework on feedback.

### CODE REVIEW

File: docs/docs/20-usage/51-plugins/10-plugins.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ pipeline:
 
 Plugins are just pipeline steps. They share the build workspace, mounted as a volume, and therefore have access to your source tree.
 
+## Finding Plugins
+
+For existing plugins there's a few places you can look:
+
+- [Official Woodpecker Plugins](https://woodpecker-ci.org/plugins)
+- [Drone Plugins](http://plugins.drone.io)
+- [The Geek Lab Drone Plugins](https://drone-plugin-index.geekdocs.de/plugins/drone-matrix/)
```
Comment (anbraten) [inline docs/docs/20-usage/51-plugins/10-plugins.md:39]: What do you think about only having the link to the official woodpecker plugin list here and adding the other two links on the plugins list page somewhere like: `There are also other plugin lists with additional plugins. Keep in mind that [Drone](https://www.drone.io/) plugins are generally supported, but could need some adjustments and tweaking.`

Comment (hay-kot) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: Refactored a bit and added a Tip section

CleanShot 2023-03-05 at 20 25 18@2x

Let me know what you think

Now re-reading what you wrote I realize that isn't what you meant 🤦

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
thanks :)

File: docs/docs/20-usage/51-plugins/10-plugins.md (hunk 1 above)
Comment (anbraten) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: @6543 This wasn't ready yet. 👆

Comment (6543) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: Hmm i did go throu your points and thought they where addressed. The wording also looked good 🤔

Comment (6543) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: is it only a smal missing bit?

Comment (anbraten) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: That's why I haven't resolved it yet 😉 

The current place is just confusing / could be missed, so the idea was to link to https://woodpecker-ci.org/plugins and place a hint about other indexes there.

Comment (6543) [reply docs/docs/20-usage/51-plugins/10-plugins.md]: Ok, i still dont get the difference to now 100%.

@hay-kot if you do, just send a pull i guess else I'll DM you and create a patch @anbraten 

Sorry for the extra work

**Pull request merged**